app/views/posts/index.html.erb

```diff
@@ -1 +1,33 @@
-<h1>Welcome to my posts</h1>
\ No newline at end of file
+<div class="wrapper">
+  <ul class="user-list">
+    <li class="user-list-item">
+      <img src=<%=  @user.photo %> alt="user">
+      <div>
+        <h3><%= @user.name %></h3>
+        <p class="float-right">Number of posts: <span><%=@user.posts_counter %></span>
+      </div>
+    </li>
+  </ul>
+   <ul class="post-list list-type-none">
+    <% @posts.each do |post| %>
+      <li class="post-list-item">
+      <div class="border post">
+        <p>Post #<%= post.id %></p>
+        <p><%= post.text %></p>
+        <ul class="comments-likes">
+          <li>Comments: <%=post.comments_counter %></li>
+          <li>Likes: <%=post.likes_counter %></li>
+        </ul>
+      </div>
+      <ul class="list-type-none border comments">
+        <% post.recent_comments.each do |comment|%>
+          <li><%= comment.user.name %><%= comment.text %></li>
+        <% end %>
```

Comment:
- [ ] As a comment belongs to an `author`, kindly make sure you also update the above line by using the `author` key word, else where you'll find another error when running your app. i.e.
```ruby
  ...
  <li><%= comment.author.name %><%= comment.text %></li>
  ...
```